src/main/java/com/algorand/algosdk/builder/transaction/ApplicationCreateTransactionBuilder.java

```diff
@@ -71,4 +73,16 @@ public T globalStateSchema(StateSchema globalStateSchema) {
         this.globalStateSchema = globalStateSchema;
         return (T) this;
     }
+
+    /**
+     * extraPages allows you to rent extra pages of memory for the application. Each page is 2048 bytes of shared
+     * memory between approval and clear state programs. extraPages parameter must be an integer between 0 and 3 inclusive.
+     */
+    public T extraPages(Long extraPages) {
+        if (extraPages < 0 || extraPages > 3) {
```

Comment:
```suggestion
        if (extraPages == null || extraPages < 0 || extraPages > 3) {
```
extraPages can be null and we'd still want the same exception to appear

Reply:
@jdtzmn good catch! Alternative suggestion: change the argument from `Long` to `long`.

Reply:
I'll go with jacob's to match the other variables which are Long